src/services/accessValidation.js

```diff
@@ -0,0 +1,25 @@
+import { User, sequelize } from '../models';
+import logger from '../logger';
+
+const namespace = 'SERVICE:ACCESS_VALIDATION';
+
+/**
+ * Finds or creates a user in the database.
+ *
+ * //potentially use also or instead of the user id provided by HSES
+ * @param {Object} userData - user information containing email address
+ * @returns {Promise<any>} - returns a promise
+ */
+export default function findOrCreateUser(data) {
+  return sequelize.transaction((transaction) => User.findOrCreate({
+    where: {
+      email: data.email,
```

Comment:
Should this be something like:
```suggestion
      ...data,
```
so the data object can have more then just the user's email address?